script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt

```diff
@@ -270,37 +282,45 @@ The seriousness of her voice makes me straighten up.
 [sha:3a485105919df7fabba06ab509aa91f37abe0013]{
 -- Page 253, Offset 1093.
 -- 貴方は先ほど、有間の家の人たちを過去形で話しました。貴方本人ももう切り離して考えている。遠野くんはそれぐらい、前のご家族が嫌いだったんですか？」
-Just now, you were talking about the Arima family in the past tense. You're already thinking of them as a separate group. Tohno-kun, did you hate your previous family that much?"
+// 前のご家族 changed to foster because at this point he has at least 2 previous families
+Just now, you were talking about the Arima family in the past tense.
+ You're already thinking of them as people you've cut yourself off from.
+ Tohno-kun, did you hate your foster family that much?"
 }
 [sha:cae08cd911c698bad23cedeaba622169da2e13a9]{
 -- Page 254, Offset 1094.
 -- 前のご家族―――この七年間、育ての親だった有間の人たちの事か。
-Does she mean the Arima family? They were my parents for seven years, after all.
+Foster family―――does the mean the Arimas?
+ I guess they're the ones who looked after me for the last seven years.
 }
 [sha:ae47ae663ce1affeda763cd6c891d9946c0599ed]{
 -- Page 254, Offset 1095.
 -- 本当の両親ではない父親と母親。
-A mother and father who were not my real parents.
+They may have been my mother and father, but they weren't my real parents.
 }
 [sha:86ebfd9f6f22b1c3970de77a5d36de06cde801d2]{
 -- Page 254, Offset 1096.
 -- 親戚の家であるはずの見知らぬ建物。
-A strange building that was supposed to be a relative's house.
+// Not 100% sure what であるはず is adding here
```

Comment:
Yeah, it just seems like a weird way to put it since like... he knows it's the Arimas' house and that they're relatives? I guess do you think it's worth trying to encode?

Reply:
not really, no. I think this line is fine as-is